CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 2.7.0-beta.1
+
+### Features ✨ and improvements 🏁
+
+* Enable preloading tiles for camera animation. ([#11328](https://github.com/mapbox/mapbox-gl-js/pull/11328))
+* Improve quality of transparent line layers by removing overlapping geometry artifacts. ([#11082](https://github.com/mapbox/mapbox-gl-js/pull/11082))
+* Add perspective correction for non-rectangular image, canvas and video sources. ([#11292](https://github.com/mapbox/mapbox-gl-js/pull/11292))
+ox-gl-js/pull/11279)) (h/t [lpizzinidev](https://github.com/lpizzinidev))
```

Comment:
This test seems lost, I think it belongs down below

Reply:
I think I fixed this one already! Committed the other changes!